modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,264 @@
+    port: 4984 <3>
+    targetPort: 4984
+  type: LoadBalancer
+----
+<1> `metadata.name`: the name of the load balancer is "sgw-load-balancer".
+<1> `spec.selector.app`: this value corresponds to the pods targeted by the load balancer.
+In this case, it targets any pods with the `app=sync-gateway` label which are the Sync Gateway nodes - this corresponds to what was specified in the deployment yaml file.
+<1> `spec.ports[].targetPort`: the load balancer service targets ports 4984 on the Sync Gateway cluster.
+This is the Sync Gateway port corresponding to the xref:rest-api.adoc[REST API].
+For security purposes, it is recommended that you do not expose the admin port (4985) over the Internet.
+. Deploy the load balancer.
++
+[source,console]
+----
+kubectl create -f sgw-load-balancer.yaml
+----
+If successful, you will see the following.
++
+[source,console]
+----
+service "sgw-load-balancer" created
+----
+. Verify the status of the service creation with the following.
++
+[source,console]
+----
+kubectl get services
+----
+If successful, you will see the following.
++
+[source,console]
+----
+NAME                TYPE           CLUSTER-IP     EXTERNAL-IP
+cb-example          ClusterIP      None           <none>
+cb-example-srv      ClusterIP      None           <none>
+cb-example-ui       NodePort       10.3.246.239   <none>
+kubernetes          ClusterIP      10.3.240.1     <none>
+sgw-load-balancer   LoadBalancer   10.3.253.17    35.184.19.17
+----
+The *sgw-load-balancer*'s `EXTERNAL-IP` is the load balancer's publicly accessible hostname.
+. Verify the pods that the load balancer is targeting.
++
+[source,console]
+----
+kubectl describe service sgw-load-balancer
+----
+You should see the equivalent of the following.
++
+[source,console]
+----
+Name:                     sgw-load-balancer
+Namespace:                default
+Labels:                   <none>
+Annotations:              <none>
+Selector:                 app=sync-gateway
+Type:                     LoadBalancer
+IP:                       10.3.253.17
+LoadBalancer Ingress:     35.184.19.17
+Port:                     <unset>  4984/TCP
+TargetPort:               4984/TCP
+NodePort:                 <unset>  32397/TCP
+Endpoints:                10.0.0.34:4984,10.0.0.35:4984
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:
+----
+Notice the "endpoints" field and confirm that it corresponds to the Sync Gateway nodes.
+In this example, we have 2 Sync Gateway nodes (two regular nodes).
+. Verify the Sync Gateway cluster is accessible with the following command; where `EXTERNAL-IP` is the IP that was copied in step 3.
++
+[source,console]
+----
+curl  http://EXTERNAL-IP:4984
+----
+It should return the following.
++
+[source,console]
+----
+{"couchdb":"Welcome","vendor":{"name":"Couchbase Sync Gateway","version":"2.1"},"version":"Couchbase Sync Gateway/2.1.1(17;fea9947)"}
+----
+
+You have successfully deployed a Sync Gateway cluster on Kubernetes.
```

Comment:
One thing I think would be nice here is a quick example of getting the Sync Gateway logs to check it's running correctly (i.e. connected to the bucket, as it might be running but not successfully connecting to Couchbase Server).

This serves 2 purposes:
1. Introduces the user to the concept of retrieving the Sync Gateway logs through Kubernetes-native APIs.
2. It ensures that the user has definitely set up Sync Gateway + CB Server correctly.

Reply:
OK. https://github.com/couchbase/docs-sync-gateway/pull/35/files#diff-b4b7021551879ebbed08d87022104170R5

Reply:
Marking this convo as resolved. Copied link to this comment on new PR.